src/main/java/Hw/Main.java

```diff
@@ -2,6 +2,56 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Домашнее задание № 9");
+        System.out.println("Задание 1");
+        int[] payOfMonday = {25000, 30000, 28000, 35000, 33000};
+        int sumOfMonth = 0;
+        for (int i = 0; i < payOfMonday.length; i++) {
+            sumOfMonth = sumOfMonth + payOfMonday[i];
+        }
+        System.out.println("Сумма трат за месяц составила " + sumOfMonth + "  рублей");
+        System.out.println("Задание 2");
+        int[] payOfDay = {35000, 30000, 29000, 31000, 33000};
+        int minOfPay = payOfDay[0];
+        int maxOfPay = payOfDay[0];
+        for (int i = 1; i < 5; i++) {
```

Comment:
не нужно использовать числа с потолка, надо использовать длину массива
```suggestion
        for (int i = 1; i < payOfDay.length; i++) {
```